### ISSUE TITLE
We hope to reproduce the performance of GPS on VTAB

### ISSUE THREAD

**Issue by dafeizhong** [FightingFighting/GPS#4]
Can you provide a fine-tuning script for VTAB?

**Comment by FightingFighting** [comment]
we will release the code shortly.

**Comment by FightingFighting** [comment]
the scripts already released

**Comment by xjiangmed** [comment]
@FightingFighting I have run the provided training scripts for the VTAB-1K benchmark, and the performance on three datasets (Cifar100, flower102, and pets) is still far from the results reported in the paper even after tweaking the hyperparameters. Notably, on the flower102 and pets datasets, the SSF method similarly yielded results of 87.70 and 82.36. Could you give me some suggestions to reproduce the results of GPS? Thank you in advance for your guidance. 
![image](https://github.com/user-attachments/assets/db62ca2b-2b68-4ba0-ae73-eb8e7bea8497)

**Comment by FightingFighting** [comment]
Hi @xjiangmed , I just update all scripts of VTAB for the hyperparameters.  As for cifar100, in order to compare with VPT, we run GPS on their code. For the flower,  is there something wrong with your code? I just rerun flowers (using the old scripts) and I got the Acc results: 99.5.

**Comment by FightingFighting** [comment]
and you can also try to change some hyperparameters for searching, such as lr, bz and times_para. Hope this could help.

**Comment by xjiangmed** [comment]
@FightingFighting Thank you for your help. I really appreciate it!

**Comment by CSYangHsu** [comment]
Hi @FightingFighting @xjiangmed,

I’m wondering if the validation dataset in the code refers to the testing dataset. Also, is the eval_top1 in summary.csv the testing accuracy reported in the paper?

Thanks!

**Comment by CSYangHsu** [comment]
Hi @FightingFighting  @xjiangmed ,

I’m wondering if the validation dataset in the code refers to the testing dataset. Also, is the eval_top1 in summary.csv the testing accuracy reported in the paper?

Thanks!
